coral/src/app/pages/index.tsx

```diff
@@ -1,9 +1,13 @@
 import Layout from "src/app/layout/Layout";
+import { PageHeader } from "@aivenio/design-system";
 
 const HomePage = () => {
   return (
     <Layout>
-      <h1>Index</h1>
+      <>
```

Comment:
Nitpick: why do we have a fragment here? Layout wrapping those two components should be enough?